cypress.config.js

```diff
@@ -0,0 +1,20 @@
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    supportFile: false,
+    specPattern: 'cypress/integration/**/*.spec.{js,ts,jsx,tsx}',
```

Comment:
The 'specPattern' seems to have a missing forward slash before the wildcard character '*.spec.{js,ts,jsx,tsx}'. It should likely be 'cypress/integration/**/*.spec.{js,ts,jsx,tsx}' to correctly match files in any subdirectories.